src/Doctrine/Orm/RedirectRoute.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm;
+
+use Symfony\Cmf\Bundle\RoutingBundle\Model\RedirectRoute as RedirectRouteModel;
+
+/**
+ * {@inheritdoc}
+ *
+ * This extends the RedirectRoute Model.
```

Comment:
this comment is not very helpful. maybe say "Provide a redirect route stored in Doctrine ORM." or something like that?